frontend/src/MuiForms/RichTextInput.tsx

```diff
@@ -40,7 +49,7 @@ export default function RichTextInput({ label, value, formContext, onChange, req
     <RichTextEditor
       value={value}
       onChange={onChange}
-      textareaProps={{ disabled: disabled }}
+      textareaProps={{ disabled: disabled, id: id }}
```

Comment:
I may have missed it, but I didn't see anywhere this `id` was being used. Could you explain why it's been added?

Reply:
This is so that it will have an `id` prop for us to call `cy.get()` with. We might be able to swap it for `data-set` but I'm wary as last time I'm sure I tried and it didn't work.

Reply:
Happy to leave as is for now then